src/engine/routing_algorithms/alternative_path_mld.cpp

```diff
@@ -641,8 +635,8 @@ makeCandidateVias(SearchEngineData<Algorithm> &search_engine_data,
                                            forward_heap,
                                            overlap_via,
                                            overlap_weight,
-                                           force_loop_forward,
-                                           force_loop_backward,
+                                           DO_NOT_FORCE_LOOPS,
+                                           DO_NOT_FORCE_LOOPS,
```

Comment:
If we never have to force loops for mld then the code path above here in the routing base mld

https://github.com/Project-OSRM/osrm-backend/pull/4563/files#diff-b397fc7de6b623ef23bec900b2bb5c27R258

can never be triggered and the routingStep interface for mld really should be changed, no?

Reply:
Ah just seeing https://github.com/Project-OSRM/osrm-backend/issues/4564 👍